src/mcedit2/editorcommands/analyze.py

```diff
@@ -0,0 +1,67 @@
+"""
+    analyze
+"""
+from __future__ import absolute_import
+from PySide import QtGui, QtCore
+import logging
+
+from mcedit2.util.load_ui import load_ui
+
+log = logging.getLogger(__name__)
+
+class AnalyzeOutputDialog(QtGui.QDialog):
+    def __init__(self, editorSession, blockCount, entityCount, tileEntityCount, *args, **kwargs):
+        super(AnalyzeOutputDialog, self).__init__(*args, **kwargs)
+        self.editorSession = editorSession
+        self.blocktypes = editorSession.worldEditor.blocktypes
+        
+        load_ui("analyze.ui", baseinstance=self)
+        blockTable = self.blockOutputTable
+        self.setupBlockTable(blockCount, blockTable)
+        
+        entityTable = self.entityOutputTable
+        self.setupEntityTable(entityCount, tileEntityCount, entityTable)
+
+        self.sizeHint()
```

Comment:
This is a no-op. `sizeHint` only returns a value and is normally called by the layout.

Correct use of layouts in the UI file should take care of making widgets line up and fill spaces correctly.